MAINTAINERS.txt

```diff
@@ -25,3 +25,8 @@ Patrick Zielinski
   Email: patrick.z@nyu.edu
   GitHub username: @patzielinski
   Affiliation: NYU
+
+Neil Naveen
+  Email: naveen.neil@icloud.com
+  GitHub username: @neilnaveen
+  Affiliation: NYU
```

Comment:
@JustinCappos can you double check the affiliation works here?

Reply:
Sure, I'm fine with however he wants to be listed.